packages-exp/auth-exp/index.ts

```diff
@@ -63,6 +63,13 @@ export { browserPopupRedirectResolver } from './src/platform_browser/popup_redir
 // MFA
 export { PhoneMultiFactorGenerator } from './src/platform_browser/mfa/assertions/phone';
 
+/**
+ * Initializes an Auth instance with default depedencies.
```

Comment:
Worth calling out that it's default dependencies for the specific platform? 
And further, since we've split the index files by platform, can we specifically say which dependencies (`browserPopupRedirectResolver`) and maybe even show how it's equivalent to an `initializeAuth` call in an `@example`?

Reply:
only index.ts is used for documentation, so I don't think so...